free-2.md

```diff
@@ -11,41 +11,135 @@ _What need is met by your idea? Who are you helping? What is that person's
 experience like now? What would their experience be like if you could help 
 them?_
 
+This language would help individuals take a more active role in managing their
+finances. Currently, small investors have a few options: invest in a large
+mutual fund and let the pros take care of it, or subscribe to an online
+brokerage site like Interactive Brokers and use their tools. There are two
+needs not met by these options: ease of use, and flexibility.
+
+To use tools like Interactive Brokers, the trader must have some technical
+knowledge, including familiarity with jargon that may seem arbitrary. For
+example, what is a limit order? What is a market order? There are two prices
+listed here; will my market order to buy clear at best bid or best ask? A more
+intuitive vocabulary could lower the technical barriers of entry to trading.
+
+Algo-trading or high-speed trading is a technique that is all but off limits to
+ordinary traders. According to Prof. Evans, most commercial trading software is
+written in C, which obviously requires a technical knowledge lacked by most
+traders. Why not have a DSL specifically designed to make trading operations
+easy, and open up this technique to more than just large corporations?
```

Comment:
There are languages which can, depending on the application and how they're written, compete with C/C++. Furthermore, it isn't that hard to imagine a DSL that compiles down to optimized C/C++.